app/src/main/res/layout/activity_main_menu.xml

```diff
@@ -45,31 +68,78 @@
     </LinearLayout>
 
 
-    <LinearLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_weight="1"
         android:orientation="horizontal">
 
```

Comment:
Button IDs should be descriptive. It's a minor thing, but they add up. Something like `buttonExpenseAdd` would work.